shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -283,6 +283,7 @@ object GlutenConfig {
 
   var GLUTEN_ENABLE_BY_DEFAULT = true
   val GLUTEN_ENABLE_KEY = "spark.gluten.enabled"
+  val GLUTEN_THREAD_LEVEL_ENABLE_KEY = "spark.gluten.thread.level.enabled"
```

Comment:
Does `spark.gluten.enabledForThisThread` look better?

Reply:
Seems to me that `spark.gluten.thread.level.enabled` is not a Spark config key. Can you move it out to another file?

Reply:
rename to `gluten.enabledForCurrentThread`